staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher_test.go

```diff
@@ -224,6 +224,12 @@ func TestWatchFromNoneZero(t *testing.T) {
 }
 
 func TestWatchError(t *testing.T) {
+	// we need to check for watch errors so do not panic the server
```

Comment:
@wojtek-t yeah, though I do not know of a better way that does not require using locks.